modules/dacIdSystem.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * This module adds dacId to the User ID module
+ * The {@link module:modules/userId} module is required
+ * @module modules/dacIdSystem
+ * @requires module:modules/userId
+ */
+
+import { submodule } from '../src/hook.js';
+import { getStorageManager } from '../src/storageManager.js';
+
+export const storage = getStorageManager();
+
+export const cookieKey = '_a1_f';
+
+export const dacIdSystemSubmodule = {
+  /**
+   * used to link submodule with config
+   * @type {string}
+   */
+  name: 'dacId',
+
+  /**
+   * performs action to obtain id
+   * @function
+   * @returns { {id: {dacId: string}} | undefined }
+   */
+  getId: function() {
+    const newId = storage.getCookie(cookieKey);
+    if (!newId) {
+      return undefined;
+    }
+    const result = {
+      dacId: newId
+    }
+    return {id: result};
```

Comment:
why not consolidate into a single `return` statement? no issue not to...just a suggestion

Reply:
Because it may return values other than id in the future.

Reply:
ok sounds good.